### PR TITLE
Change vendor from Eclipse Foundation to Eclipse Adoptium

### PR DESCRIPTION
**Pull request by sxa**
As discussed in the marketing group meeting and in the Adoptium Workgroup Steering Committee meetings this week. This should remove ambiguity to end users by clarifying that this is the "Adoptium" build.

Signed-off-by: Stewart X Addison <sxa@redhat.com>

### CODE REVIEW

**Review by karianna — APPROVED** [review]
LGTM

**Comment by hendrikebbers** [comment]
LGTM

File: sbin/build.sh [hunk 1]
```diff
@@ -199,9 +199,10 @@ getOpenJdkVersion() {
 
 # Ensure that we produce builds with versions strings something like:
 #
-# openjdk version "1.8.0_131"
-# OpenJDK Runtime Environment (build 1.8.0-temurin-<user>_2017_04_17_17_21-b00)
-# OpenJDK 64-Bit Server VM (build 25.71-b00, mixed mode)
+# openjdk 11.0.12 2021-07-20
+# OpenJDK Runtime Environment Temurin-11.0.12+7 (build 11.0.12+7)
+# OpenJDK 64-Bit Server VM Temurin-11.0.12+7 (build 11.0.12+7, mixed mode)configureVersionStringParameter() {
```
Comment (jerboaa) [inline sbin/build.sh:204]: Is this trailing `configureVersionStringParameter() {` intentional?

Comment (sxa) [reply sbin/build.sh:204]: Ah that explains something I hit earlier - no it's not - will fix - well spotted :-)

**Review by jerboaa — APPROVED** [review]

**Comment by tushev** [comment]
Could you please tell where exactly `vendor` will be changed - in API, in `release` file, in default installation paths, registry paths?

**Comment by karianna** [comment]
IIRC at this stage it is only:

* The `vendor` property in the json meta data - OpenJDK17-jdk_x64_windows_hotspot_17_35.msi.json
* `java.vendor` Java system property 

We're looking at whether we need to change the installer paths as well.  More on this shortly

**Comment by tushev** [comment]
@karianna Thanks!

**Pull request merged**